modules/accredited_representative_portal/spec/factories/power_of_attorney_form.rb

```diff
@@ -1,10 +1,60 @@
 # frozen_string_literal: true
 
+form_data = <<~JSON
```

Comment:
Can you simplify this and improve readability by using a Hash structure directly in your factory and converting it to JSON as needed?